lib/active_merchant/billing/gateways/vantiv_express.rb

```diff
@@ -0,0 +1,475 @@
+require 'nokogiri'
+require 'securerandom'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class VantivExpressGateway < Gateway
+      self.test_url = 'https://certtransaction.elementexpress.com'
+      self.live_url = 'https://transaction.elementexpress.com'
+
+      self.supported_countries = ['US']
+      self.default_currency = 'USD'
+      self.supported_cardtypes = %i[visa master american_express discover diners_club jcb]
+
+      self.homepage_url = 'http://www.elementps.com'
+      self.display_name = 'Element'
+
+      SERVICE_TEST_URL = 'https://certservices.elementexpress.com'
+      SERVICE_LIVE_URL = 'https://services.elementexpress.com'
+
+      NETWORK_TOKEN_TYPE = {
```

Comment:
Does this gateway support third party Network Tokens?

Reply:
According to the `wallet_type` enumeration that this is being used to fill they only support apple_pay, google_pay, and samsung_pay.